compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -1950,7 +1950,6 @@ object SymDenotations {
 
     /** The this-type depends on the kind of class:
      *  - for a package class `p`:  ThisType(TypeRef(Noprefix, p))
-     *  - for a module class `m`: A term ref to m's source module.
```

Comment:
Removed because it has not been true since dc02760eac04accb9e0d5e34128f4c79cfa8c327